package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "./src/index.js",
   "scripts": {
-    "start": "cross-env NODE_ENV=development nodemon --exec babel-node ./src/index.js",
+    "start": "cross-env NODE_ENV=development nodemon -L --exec babel-node ./src/index.js",
     "migrate": "npx sequelize-cli db:migrate",
```

Comment:
Since the commands to run the app in docker seem to be long, you can add a docker script here that will take care of everything instead of doing it manually.